package.json

```diff
@@ -95,6 +95,7 @@
     "is-https": "^3.0.0",
     "js-cookie": "^2.2.1",
     "klona": "^2.0.4",
+    "lodash.merge": "4.6.2",
```

Comment:
should be a semantic range since the rules are different in dependencies.

But I've done the change on master and released a fix already. Thanks for that though.